pkg/operator/ceph/nfs/controller.go

```diff
@@ -194,6 +195,10 @@ func (r *ReconcileCephNFS) reconcile(request reconcile.Request) (reconcile.Resul
 	// Populate CephVersion
 	currentCephVersion, err := cephclient.LeastUptodateDaemonVersion(r.context, r.clusterInfo, opconfig.MonType)
 	if err != nil {
+		if strings.Contains(err.Error(), opcontroller.UninitializedCephConfigError) {
+			logger.Info("skipping reconcile since operator is still initializing")
+			return opcontroller.ImmediateRetryResult, nil
```

Comment:
Shouldn't we delay the requeue instead of the immediate retry? If ceph isn't initialized yet, I thought we had a pattern of 10s or similar to delay.